localstack/services/s3/provider.py

```diff
@@ -1088,21 +1088,12 @@ def _fix_tagging_from_xml(fn, *args, **kwargs) -> Dict[str, str]:
         return tags
 
     @patch(moto_s3_responses.S3Response.is_delete_keys)
-    def s3_response_is_delete_keys(fn, self, request, path, bucket_name):
+    def s3_response_is_delete_keys(fn, self):
         """
         Old provider had a fix for a ticket, concerning 'x-id' - there is no documentation on AWS about this, but it is probably still valid
         Additional fix for testing if this is a "delete_objects" request, by removing trailing "=" from the path
```

Comment:
The docs about "Additional fix ..." is now obsolete and should be removed. 
But it would be great if we kept a comment referencing the issue (#3931) that requires this patch